### PR TITLE
Fix wrongly placed block end tag in carts/helpers/view/view.tpl

### LINKED ISSUE #20286: Small formal bug in carts/helpers/view/view.tpl
imho it is only formal bug and cannot cause a problem 

the file **admin/themes/default/template/controllers/carts/helpers/view/view.tpl**

ps17
 
end of block inside of inner div can be  confusing

### PR DESCRIPTION
**Pull request by Sinepel**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Invert a `{/block}` tag and `</div>`
| Type?         | bug fix 
| Category?     |  BO 
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #20286 
| How to test?  | Go to a cart detail in BO

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20289)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Comment by atomiix** [comment]
@PrestaShop/prestashop-core-developers does that even needs QA?

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
@atomiix QA is needed, when we change an HTML tag we must be sure everything is ok everywhere :sweat_smile:

File: admin-dev/themes/default/template/controllers/carts/helpers/view/view.tpl [hunk 1]
```diff
@@ -220,5 +220,5 @@
 	<div class="row alert alert-warning">
 		{l s='For this particular customer group, prices are displayed as:' d='Admin.Orderscustomers.Notification'} <b>{if $tax_calculation_method == $smarty.const.PS_TAX_EXC}{l s='Tax excluded' d='Admin.Global'}{else}{l s='Tax included' d='Admin.Global'}{/if}</b>
 	</div>
-{/block}
 </div>
+{/block}
```
Comment (PierreRambaud) [inline admin-dev/themes/default/template/controllers/carts/helpers/view/view.tpl]:
```suggestion
{/block}

```

Comment (NeOMakinG) [inline admin-dev/themes/default/template/controllers/carts/helpers/view/view.tpl]: Hey @Sinepel, thanks for you'r contribution.

Please configure your IDE so it add new lines at the end of files !

**Review by NeOMakinG — DISMISSED** [review]

File: admin-dev/themes/default/template/controllers/carts/helpers/view/view.tpl (hunk 1 above)
Comment (Sinepel) [reply admin-dev/themes/default/template/controllers/carts/helpers/view/view.tpl]: @NeOMakinG  Hi, no problem, thank you!

**Review by Progi1984 — DISMISSED** [review]

**Comment by matthieu-rolland** [comment]
**EDIT:**
As seen with @Sinepel this PR will remain on develop, I'll change the issue too.

**Review by matthieu-rolland — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Thanks for your PR @Sinepel , it's QA :heavy_check_mark: :)

**Pull request merged**